hack/tools/boskosctl/main.go

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
Maybe we should consider introducing a separate go module in hack/tools

While I was able to avoid introducing new dependencies, it would probably allow us to use a lower minimum Go version.

Which then makes it easier to consume it from other repos/branches that still use an older go version

(have to explore this a bit)

Probably the easier alternative is to simply push an image with the boskosctl binary 

In any case. I don't want to have to backport changes to the janitor or boskosctl because older branches use their own copy of it

Reply:
Implemented the Docker image